src/components/pages/order/adminPanel/BlocTabs.js

```diff
@@ -40,7 +38,11 @@ export default function BlocTabs() {
               label={tab.label}
               className={tab.className}
               Icon={tab.Icon}
-              onClick={tab.onClick}
+              onClick={
```

Comment:
```suggestion
              onClick={selectTab}
```

créé selectTab et regroupe tout ça là haut pour faire encore plus clean 😉  mais sinon, superbe, c'est encore plus malin que ce que moi j'avais trouvé 😳

Reply:
merci @vileaddev ça fait plaisir !!